pkg/authenticator/authenticator.go

```diff
@@ -28,8 +29,17 @@ import (
 )
 
 const (
+	// ProviderType can be VPC/Bluemix
 	ProviderType string = "ProviderType"
-	SecretKey    string = "SecretKey"
+
+	// SecretKey ...
+	SecretKey string = "SecretKey"
+
+	// maxRetryAttempt ...
+	maxRetryAttempt = 15
+
+	// maxRetryGap ...
```

Comment:
Is there any reason for using the retrygap as  8, 4 ?
4 -> 1st attempt
8 -> 14 attempts 
Total of 116 seconds?

Reply:
Yes. will discuss with team regarding the ideal number. My thought was, retrying for 2 mins. We can try upto 5 mins. We can discuss on this

Reply:
Upto 1 min, keep exponential retry. 2, 4., 8, 16 ...